R/shell.R

```diff
@@ -57,6 +57,16 @@ make_flowr_session_storage <- function() {
             break
           }
         }
+
+        status <- sys::exec_status(pid, FALSE)
+        if (!is.na(status)) {
+          message(paste0("[flowR] There was an error starting the flowR server, and it exited with code ", status, ". If this keeps happening, you may need to reinstall flowR by running the \"Install Node.js and flowR Shell\" addin (rstudioaddinflowr:::install_node_addin()) again."))
+          if (rstudioapi::showQuestion("[flowR] Server error", "There was an error starting the flowR server. If this keeps happening, you may need to reinstall flowR. Do so now?")) {
+            install_node_addin()
```

Comment:
This does not a full reinstall does it? In the terms of removing preexisting downloaded files?

Reply:
It does! The first step in `install_node_addin` is clearing the `_node` ordner, which hosts both the node installation and the downloaded node modules (namely flowr).